components/resources/commands.grdp

```diff
@@ -0,0 +1,487 @@
+  </message>
+  <message name="IDS_IDC_SHOW_ADDRESSES">
+    Show Addresses
+  </message>
+  <message name="IDS_IDC_EDIT_MENU">
+    Edit Menu
+  </message>
+  <message name="IDS_IDC_FIND_NEXT">
+    Find Next
+  </message>
+  <message name="IDS_IDC_FIND_PREVIOUS">
+    Find Previous
+  </message>
+  <message name="IDS_IDC_CLOSE_FIND_OR_STOP">
+    Close Find Or Stop
+  </message>
+  <message name="IDS_IDC_FIND_MENU">
+    Find Menu
+  </message>
+  <message name="IDS_IDC_ZOOM_PLUS">
+    Zoom In
+  </message>
+  <message name="IDS_IDC_ZOOM_NORMAL">
+    Reset Zoom
+  </message>
+  <message name="IDS_IDC_ZOOM_MINUS">
+    Zoom Out
+  </message>
+  <message name="IDS_IDC_ZOOM_PERCENT_DISPLAY">
+    Show current zoom
+  </message>
+  <message name="IDS_IDC_FOCUS_TOOLBAR">
+    Focus Toolbar
+  </message>
+  <message name="IDS_IDC_FOCUS_LOCATION">
+    Focus Location
+  </message>
+  <message name="IDS_IDC_FOCUS_SEARCH">
+    Focus Search
+  </message>
+  <message name="IDS_IDC_FOCUS_MENU_BAR">
+    Focus Menu Bar
+  </message>
+  <message name="IDS_IDC_FOCUS_NEXT_PANE">
+    Focus Next Pane
+  </message>
+  <message name="IDS_IDC_FOCUS_PREVIOUS_PANE">
+    Focus Previous Pane
+  </message>
+  <message name="IDS_IDC_FOCUS_BOOKMARKS">
+    Focus Bookmarks
+  </message>
+  <message name="IDS_IDC_FOCUS_INACTIVE_POPUP_FOR_ACCESSIBILITY">
+    Focus Inactive Popup For Accessibility
+  </message>
+  <message name="IDS_IDC_FOCUS_WEB_CONTENTS_PANE">
+    Focus Web Contents Pane
+  </message>
+  <message name="IDS_IDC_SHOW_BOOKMARK_BAR">
+    Show Bookmark Bar
+  </message>
+  <message name="IDS_IDC_SHOW_HISTORY">
+    Show History
+  </message>
+  <message name="IDS_IDC_SHOW_BOOKMARK_MANAGER">
+    Show Bookmark Manager
+  </message>
+  <message name="IDS_IDC_IMPORT_SETTINGS">
+    Import Settings
+  </message>
+  <message name="IDS_IDC_EDIT_SEARCH_ENGINES">
+    Edit Search Engines
+  </message>
+  <message name="IDS_IDC_HELP_PAGE_VIA_KEYBOARD">
+    Show Help
+  </message>
+  <message name="IDS_IDC_SHOW_APP_MENU">
+    Show App Menu
+  </message>
+  <message name="IDS_IDC_DEV_TOOLS_INSPECT">
+    Dev Tools Inspect
+  </message>
+  <message name="IDS_IDC_UPGRADE_DIALOG">
+    Upgrade Brave
+  </message>
+  <message name="IDS_IDC_SHOW_AVATAR_MENU">
+    Show Avatar Menu
+  </message>
+  <message name="IDS_IDC_DEV_TOOLS_TOGGLE">
+    Dev Tools Toggle
+  </message>
+  <message name="IDS_IDC_SHOW_SETTINGS_RESET_BUBBLE">
+    Reset Settings
+  </message>
+  <message name="IDS_IDC_SHOW_SRT_BUBBLE">
+    Show SRT Bubble
+  </message>
+  <message name="IDS_IDC_TOGGLE_FULLSCREEN_TOOLBAR">
+    Toggle Fullscreen Toolbar
+  </message>
+  <message name="IDS_IDC_CUSTOMIZE_TOUCH_BAR">
+    Customize Touch Bar
+  </message>
+  <message name="IDS_IDC_SHOW_BETA_FORUM">
+    Show Beta Forum
+  </message>
+  <message name="IDS_IDC_TOGGLE_JAVASCRIPT_APPLE_EVENTS">
+    Toggle Javascript Apple Events
+  </message>
+  <message name="IDS_IDC_INSTALL_PWA">
+    Install Pwa
```

Comment:
```suggestion
    Install PWA
```

Reply:
Same with VPN; just make sure all occurrences of PWA are using the capitalized one 😄